lib/validates_zipcode/cldr_regex_collection.rb

```diff
@@ -132,117 +134,120 @@ module CldrRegexpCollection
       PF: /\A987\d{2}\z/,
       PG: /\A\d{3}\z/,
       PM: /\A9[78]5\d{2}\z/,
-      PN: /\APCRN 1ZZ\z/,
+      PN: /\APCRN[ ]?1ZZ\z/i,
       PW: /\A96940\z/,
-      RE: /\A9[78]4\d{2}\z/,
-      SH: /\A(ASCN|STHL) 1ZZ\z/,
+      RE: /\A((97)(4|7|8)[0-9][0-9])\z/,
```

Comment:
Looking at https://en.wikipedia.org/wiki/Postal_codes_in_France it seems to give a list for all the territories e.g. Reunion: this _should_ be restricted to just `/\A974\d{2}\z/`

Let me know what you think